frontend/src/components/accounts/AccountDetails.tsx

```diff
@@ -119,9 +119,13 @@ export default class extends React.Component<Props, State> {
             <CardCell
               title="Creation Hash"
               text={
-                <TransactionLink transactionHash={account.address}>
-                  {account.address}
-                </TransactionLink>
+                account.address === "" ? (
+                  ""
```

Comment:
Should we display "from genesis" here?

Reply:
maybe not, let's wait to show real information about genesis when we finish fetching from genesis?

Reply:
Ok. Fair enough.